update-dois/src/main/java/org/reactome/release/updateDOIs/Main.java

```diff
@@ -12,55 +14,57 @@
 //  final static Logger logger = Logger.getLogger(UpdateDOIs.class);
 	private static final Logger logger = LogManager.getLogger();
 
-  public static void main( String[] args ) {
+  public static void main( String[] args ) throws IOException {
 
 	 // Default locations of properties and pre-set report files
 	 // Will override if arguments are provided
-	 // TODO: Lots of room for error here if only 1 argument provided, or if the report file isn't updated
-    String pathToConfig = "src/main/resources/config.properties";
-    String pathToReport = "src/main/resources/UpdateDOIs.report";
-    if (args.length > 0 && !args[0].equals(""))
-    {
-      pathToConfig = args[0];
+    String pathToConfig = args.length > 0 && !args[0].isEmpty() ? args[0] : Paths.get("src/main/resources/config.properties").toString();
+
+    String pathToReport = Paths.get("src/main/resources/UpdateDOIs.report").toString();
```

Comment:
Be sure to separate each part of the directory path into individual parameters passed to `Paths.get()`:

E.g. `Paths.get("src", "main", "resources", "config.properties")`

That way `Paths.get()` defers to the OS for the path separator and it doesn't need hard-coding.